### ISSUE TITLE
FullName doesn’t work with multi-word surnames

### ISSUE THREAD

**Issue by RicLeP** [michael-rubel/laravel-value-objects#11]
My surname is ‘Le Poidevin’ but this gets changed to just ‘Poidevin’.

```
$name = new FullName('Richard Le Poidevin');

dd($name->toArray());

//
array:3 [
  "fullName" => "Richard Le Poidevin"
  "firstName" => "Richard"
  "lastName" => "Poidevin" // should be: Le Poidevin
]
```

It’s a tricky one, where should you break, always the first space? Some people may have multiple first names but I think that’s more likely to be hyphenated whilst surnames with ‘Le’ etc. are pretty common. It’s still better than what usually happens to my name - ‘Richard L’ 😅

Maybe the default behaviour should be `firstName` is the first word and everything else forms `lastName`?

**Comment by michael-rubel** [comment]
Full names in some countries contain multiple names, e.g. Polish "Anna Ewa Kowalska", so we can't break by the first space. I thought of adding the second optional parameter to allow passing the last name as a composite, but not sure how useful that would be.

```php
new FullName('Richard', 'Le Poidevin'); 
```

**Comment by michael-rubel** [comment]
Likely fixed in #17 🙂
